connectorx-python/connectorx/__init__.py

```diff
@@ -377,3 +385,11 @@ def reconstruct_pandas(df_infos: Dict[str, Any]):
     )
     df = pd.DataFrame(block_manager)
     return df
+
+
+def remove_ending_semicolon(query: str) -> str:
```

Comment:
This looks a bit cumbersome. Could be rewritten to:

```python
if query.endswith(';'):
    query = query[:-1]
return query
```

Reply:
Thanks @surister , indeed your suggestion is better. Can you open a PR for this?

Reply:
You got it.